doc/installation.rst

```diff
@@ -0,0 +1,56 @@
+.. Copyright (c) 2008-2016 OpenShot Studios, LLC
```

Comment:
```suggestion
.. Copyright (c) 2008-2020 OpenShot Studios, LLC
```
Might as well be current for NEW files. :wink:

Reply:
It is merely renamed to be pendantic :). But yes, updated.

Reply:
Ah, I didn't actually realize that. Still, new name new year, why not? :laughing: